VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs

```diff
@@ -358,13 +358,26 @@ public ValueTask<string> RenderTemplateAsync(string templateContent, string temp
 
                 JObject result;
                 var baseThemeSettings = new JObject();
-                var currentThemeSettings = result = GetCurrentSettingsPreset(InnerGetAllSettings(_themeBlobProvider, CurrentThemeSettingPath));
+                var allCurrentThemeSettings = InnerGetAllSettings(_themeBlobProvider, CurrentThemeSettingPath);
+                var currentThemeSettings = result = null;
+                try
+                {
+                    currentThemeSettings = result = GetCurrentSettingsPreset(allCurrentThemeSettings);
```

Comment:
The currentThemeSettings will be null when child settings file hasn't presets. It will cause the problem. see comment below.